docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-18_removed-cmatt.txt

```diff
@@ -0,0 +1 @@
+* The class ```CMAttribute``` has been removed from :mod:`iris.fileformats.rules.py`.
```

Comment:
You don't need `.py` on the end of the module definition.

Reply:
okie dokie.  Just give me a minute then...